doc/Manuals/LoadBalancing/ig-xlb_x-road_external_load_balancer_installation_guide.md

```diff
@@ -269,9 +269,7 @@ In order to properly set up the data replication, the slave nodes must be able t
 
```

Comment:
The load balancer installation guide version history needs to be updated since we are making significant changes to it

Reply:
Updated the version history